common/test.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <debug.h>
+#include <test.h>
+
+#if TEST
+
+void
+run_tests(void){
```

Comment:
Please put this function at the bottom (so this compiles), and run `make format`, and ensure there is a blank line between the function and the `#endif`

Reply:
done